frontend/src/pages/WhatIsHealthEquity/EquityTab.tsx

```diff
@@ -434,7 +434,7 @@ function EquityTab() {
               <FaqSection />
             </Grid>
           </Grid>
-          <Grid item className={styles.JoinTheEffortHeaderRow}>
+          <Grid item className={styles.JoinTheEffortHeaderRow} id="join">
```

Comment:
Is it possible to use the WIHE_JOIN_THE_EFFORT_SECTION_ID variable instead of the string "join" here?